src/Components/CreateImageWizardV2/CreateImageWizard.tsx

```diff
@@ -159,6 +160,9 @@ const CreateImageWizard = () => {
           >
             <RegistrationStep />
           </WizardStep>
+            <WizardStep name="Details" id="wizard-details">
+                <DetailsStep />
+            </WizardStep>
```

Comment:
The `CustomWizardFooter` needs to be added even when we don't need to disable the Next button, because it switches the order of the buttons.

Also suggesting changing the id to "step-details" so it's consistent with the other steps ;) (AWS and GCP are a substep, that's why the name is different)

```suggestion
          <WizardStep
            name="Details"
            id="step-details"
            footer={<CustomWizardFooter disableNext={false} />}
          >
            <DetailsStep />
          </WizardStep>
```